docs/setup_gdb.md

```diff
@@ -8,520 +8,394 @@ GDB Tutorial
+$ gdb -tui main.exe
 ```
 
 | `b main` | breakpoint on main function |
-| `b 29` | breakpoint on line 29 of current file |
-| `b stats.cpp:16` | breakpoint on line 16 of stats.cpp |
+| `b 13` | breakpoint on line 13 of current file |
+| `b main.cpp:13` | breakpoint on line 13 of main.cpp |
 | `r` | run or rerun |
-| `s` | step into |
+| `l` | list a few lines of code |
 | `n` | step over (AKA next) |
-| `refresh` | refresh TUI view |
+| `s` | step into |
+| `up` | step out |
+| `c` | continue |
 | `p myvar` | print variable `myvar` |
+| `refresh` | refresh TUI view |
+| `bt` | backtrace, useful for segfaults |
 | `q` | quit |
 
 
-# Prerequisites
-This tutorial uses command line tools.  If you haven't installed CLI tools on your machine yet, follow one of these tutorials first.
-
-| [macOS](setup_macos.html)| [Windows](setup_wsl.html) | [Linux](setup_wsl.html#install-cli-tools)
-
-We presume you've created a folder for your project, downloaded and unpacked the starter files, and created any new files with appropriate function stubs.
-
-```console
-$ pwd
-/Users/awdeorio/src/eecs280/p1-stats
-$ ls
-Makefile     main_test.in           p1_library.cpp  stats.h
-launch.json  main_test.out.correct  p1_library.h    stats_public_test.cpp
-main.cpp     main_test_data.tsv     stats.cpp       stats_tests.cpp
-```
-
-You should have function stubs in `stats.h` and `main.cpp`.  If you simply want to download a copy with the stubs already in place, use these commands.
-```console
-$ wget --no-clobber https://eecs280staff.github.io/tutorials/stats.cpp -O stats.cpp
-$ wget --no-clobber https://eecs280staff.github.io/tutorials/main.cpp -O main.cpp
-```
-
-If you're not sure how to do this, you might find one of our IDE tutorials helpful. Walk through the tutorial up through the "Create a project" section.
-
-| [VS Code](setup_vscode.html) | [Visual Studio](setup_visualstudio.html) | [Xcode](setup_xcode.html) | [Emacs](setup_emacs.html) |
-
-
-
-# Install
-GDB works best on Linux, including WSL.  If you're running macOS, use [LLDB](setup_lldb.html) instead.  If you're on CAEN Linux, GDB and GCC are already installed.  Your versions might be different.
+## Install
+You should already have `g++` and `gdb` installed from the [WSL tutorial](setup_wsl.html).  Your versions might be different.
 ```console
-$ sudo apt-get install g++ gdb
 $ g++ --version
 g++ (Ubuntu 9.3.0-17ubuntu1~20.04) 9.3.0
 $ gdb --version
 GNU gdb (Ubuntu 9.2-0ubuntu1~20.04) 9.2
 ```
 
+## Compile and Run
+GDB uses an executable you build at the command line.
 
-# Compile
-GDB debugs a compiled executable.  Compile the executable you plan to debug.
+First, compile and run your executable at the command line.
 ```console
-$ pwd
-/Users/awdeorio/src/eecs280/p1-stats
-$ make clean
-rm -rvf *.exe *~ *.out *.dSYM *.stackdump
-$ make stats_tests.exe
-g++ -Wall -Werror -pedantic -g --std=c++11 stats_tests.cpp stats.cpp p1_library.cpp -o stats_tests.exe
+$ make main.exe
```

Comment:
```suggestion
$ make main.exe
g++ -Wall -Werror -pedantic -g --std=c++11 main.cpp -o main.exe
```
I know this command is already shown in the Quick Start snippet above, but I think it's just nice to have the command there for people following along.